app/javascripts/deploy_my_codes/controllers/add_project_controller.js

```diff
@@ -0,0 +1,40 @@
+require.register('deploy_my_codes/controllers/add_project_controller', function(exports, require, module){
+  module.exports = function($scope, OrganizationService, ProjectService, UserService) {
+    var bindOrganizations = function(organizations) {
+      $scope.organizations = organizations;
+    };
+
+    var bindUser = function(user) {
+      $scope.user = user;
+    };
+
+    $scope.projects = [];
+    $scope.selectContext = function(context) {
+      _.each($scope.organizations, function(organization) { organization.selected = false });
```

Comment:
Nope. It would be a non sense. It's a page to add a project, so you select your organization and then the project. You can't select 3 organizations and 4 projects.